src/app/components/hooks/use-model-state.test.ts

```diff
@@ -84,7 +84,7 @@ describe("useModelState", () => {
 
     expect(result.current.outputState).toEqual(testState.outputState);
     act(() => {
-      result.current.setOutputState({sugarCreated: OutputAmount.Low});
+      result.current.setOutputState({sugarCreated: 0});
```

Comment:
This is an example location where you could use `OutputAmountValue.Low` instead of just `0`.